src/panels/config/integrations/integration-panels/zwave/zwave-migration.ts

```diff
@@ -79,259 +95,275 @@ export class ZwaveMigration extends LitElement {
+                        Continue
+                      </mwc-button>
+                    </div>
+                  </ha-card>
+                `
+              : ``}
+            ${this._step === 1
+              ? html`
+                  <ha-card class="content" header="Stop Z-Wave Network">
+                    <div class="card-content">
+                      <p>
+                        We need to stop the Z-Wave network to perform the
+                        migration. Home Assistant will not be able to control
+                        Z-Wave devices while the network is stopped.
+                      </p>
+                      ${Object.values(this.hass.states)
+                        .filter(
+                          (entityState) =>
+                            computeStateDomain(entityState) === "zwave" &&
+                            entityState.state !== "ready"
+                        )
+                        .map(
+                          (entityState) =>
+                            html`<ha-alert alert-type="warning">
+                              Device ${computeStateName(entityState)}
+                              (${entityState.entity_id}) is not ready yet! For
+                              the best result, wake the device up if it is
+                              battery powered and wait for this device to become
+                              ready.
+                            </ha-alert>`
+                        )}
+                      ${this._stoppingNetwork
+                        ? html`
+                            <div class="flex-container">
+                              <ha-circular-progress
+                                active
+                              ></ha-circular-progress>
+                              <div><p>Stopping Z-Wave Network...</p></div>
+                            </div>
+                          `
+                        : ``}
+                    </div>
+                    <div class="card-actions">
+                      <mwc-button @click=${this._stopNetwork}>
+                        Stop Network
+                      </mwc-button>
+                    </div>
+                  </ha-card>
+                `
+              : ``}
+            ${this._step === 2
+              ? html`
+                  <ha-card class="content" header="Set up Z-Wave JS">
+                    <div class="card-content">
+                      <p>Now it's time to set up the Z-Wave JS integration.</p>
+                      ${isComponentLoaded(this.hass, "hassio")
+                        ? html`
+                            <p>
+                              Z-Wave JS runs in a Home Assistant addon that will
+                              be setup next. Make sure to check the checkbox for
+                              the addon.
+                            </p>
+                          `
+                        : html`
+                            <p>
```

Comment:
Let's make this conditional to if `hass.config.components.includes("hassio")`

Reply:
It already is